### ISSUE TITLE
8.1.2 does not launch

### ISSUE THREAD

**Issue by kadorito** [BobStrogg/teslacam-browser#48]
Hi

I installed 8.1.2 as a replacement of 0.955 (funny versioning) and it does not want to launch on my mac.

**Comment by Bioguy** [comment]
Same as kadorito.  Crashes immediately (log attached).


[teslacam_log.rtf.zip](https://github.com/BobStrogg/teslacam-browser/files/4313472/teslacam_log.rtf.zip)

**Comment by BobStrogg** [comment]
Will investigate 👍🏻. Versioning was continuing from last manually updated version (I think, but maybe was mistaken).

As a temporary workaround it should be possible to clone the repository and run it from source... Not as convenient as the binary of course 😕

**Comment by BobStrogg** [comment]
Fixed, and reverted to previous versioning sequence.

**Comment by kadorito** [comment]
Still not working with 0.0.956

**Comment by Bioguy** [comment]
It opens just fine for me now, although for each set of files only one or two of the views (rear, left side, etc.) actually show up.  The others are just blank squares.

Then again, this is an improvement, since I had been unable to use the software at all after they switched to HEVC.  Before this version none of the videos played at all.

**Comment by BobStrogg** [comment]
I’ve noticed some views take a while to show up on Chrome (and in the app since it uses Chrome). Try sliding the scrubber forward & back a few times and see if it helps.

Currently the most reliable behavior is through Safari. Edge used to be good too, but they recently moved to Chromium 😬

**Comment by BobStrogg** [comment]
> Still not working with 0.0.956

What’s not working? Missing / blank HEVC videos are a known issue with Electron / Chromium, currently without a viable fix. Workaround is to click the “Browse” button in the app, and open the URL in Safari or an older Edge version (latest Edge is based on Chromium and probably won’t work).

**Comment by kadorito** [comment]
When I said still not working, I mean the app does not launch.

**Comment by Bioguy** [comment]
I do use Safari, but with very inconsistent results.  Recently I've switched over to using TeslaCam Video WebApp Player because it doesn't seem to have this problem for whatever reason.  

I appreciate your continued work on this project and I feel it has advantages over other similar products.

**Comment by BobStrogg** [comment]
> I do use Safari, but with very inconsistent results. Recently I've switched over to using TeslaCam Video WebApp Player because it doesn't seem to have this problem for whatever reason.
> 
> I appreciate your continued work on this project and I feel it has advantages over other similar products.

Yep this app will probably always be hobbled by the HEVC issue, short of somehow doing a custom build of Chromium with the codec installed. I’m hoping Tesla will have an in-car viewer long before then! 👍🏻
